trino/client.py

```diff
@@ -397,12 +435,14 @@ def __init__(
         port: int,
         client_session: ClientSession,
         http_session: Any = None,
-        http_scheme: str = None,
+        http_scheme: Optional[str] = None,
         auth: Optional[Any] = constants.DEFAULT_AUTH,
         redirect_handler: Any = None,
         max_attempts: int = MAX_ATTEMPTS,
-        request_timeout: Union[float, Tuple[float, float]] = constants.DEFAULT_REQUEST_TIMEOUT,
-        handle_retry=_RetryWithExponentialBackoff(),
+        request_timeout: Union[
+            float, Tuple[float, float]
+        ] = constants.DEFAULT_REQUEST_TIMEOUT,
```

Comment:
The constant is a plain float. Where is the Tuple[float, float] coming from?

Reply:
changed type hint to `float` only